README.md

```diff
@@ -19,7 +19,7 @@ Dependencies in Ubuntu/Debian:
 Build with a custom install prefix:
 
     mkdir build && cd build
-    cmake .. -DCMAKE_INSTALL_PREFIX=/home/$USER/.local
```

Comment:
Looks like you accidentally branched from your other PR?

Reply:
Yep, just caught that :)